grpc-server/grpc_server.go

```diff
@@ -43,16 +43,15 @@ type server struct {
 }
 
 // SetupGRPC setup and return a gRPC server
-func SetupGRPC(ctx context.Context, log log.Logger, facility string, errCh chan<- error) ([]byte, time.Time) {
+func SetupGRPC(ctx context.Context, log log.Logger, facility string, db *db.TinkDB, errCh chan<- error) ([]byte, time.Time) {
 	params := []grpc.ServerOption{
 		grpc.UnaryInterceptor(grpc_prometheus.UnaryServerInterceptor),
 		grpc.StreamInterceptor(grpc_prometheus.StreamServerInterceptor),
 	}
 	logger = log
 	metrics.SetupMetrics(facility, logger)
-	tinkDB := db.Connect(logger)
 	server := &server{
-		db:      tinkDB,
+		db:      db,
```

Comment:
where did `tinkDB` go/come-from? Was it a global variable? If so why can't it be deleted? Also, this change should be its own commit.

Reply:
`func SetupGRPC(ctx context.Context, log log.Logger, facility string, db *db.TinkDB, errCh chan<- error)` it is coming from `main.go`. I had to move it there in order to be able to run the migration during the tink-server's start.

Reply:
oh no nm. I thought it was a global var, but only because I failed to see `tinkDB := db.Connect` literally 2 lines up :p